### DISCUSSION TITLE
功能询问

### DISCUSSION THREAD

**Discussion by mobeicanyue** [the1812/Bilibili-Evolved#4102] [提问 / Q&A] [answered]
我想知道，预览版除了能体验到最新的功能以外，功能的数量是否和正式版一样，因为看着正式版的日志好像会更多一点

**Comment by Geeyun-JY3** [comment, marked as answer]
https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.0-preview
> 包含 v2.7.0 的所有更新内容.
